diff -ur olive/effects/internal/texteffect.cpp olive_new/effects/internal/texteffect.cpp
--- olive/effects/internal/texteffect.cpp	2020-04-15 19:22:06.000000000 +0200
+++ olive_new/effects/internal/texteffect.cpp	2020-10-06 23:17:32.574871711 +0200
@@ -32,6 +32,7 @@
 #include <QWidget>
 #include <QtMath>
 #include <QMenu>
+#include <QPainterPath>
 
 #include "ui/labelslider.h"
 #include "ui/collapsiblewidget.h"
diff -ur olive/effects/internal/timecodeeffect.cpp olive_new/effects/internal/timecodeeffect.cpp
--- olive/effects/internal/timecodeeffect.cpp	2020-04-15 19:22:06.000000000 +0200
+++ olive_new/effects/internal/timecodeeffect.cpp	2020-10-06 23:18:10.433974758 +0200
@@ -32,6 +32,7 @@
 #include <QWidget>
 #include <QtMath>
 #include <QDebug>
+#include <QPainterPath>
 
 #include "ui/labelslider.h"
 #include "ui/collapsiblewidget.h"
