--- opencolorio/src/core/CMakeLists.txt-orig	2022-04-06 12:09:20.219705597 -0400
+++ opencolorio/src/core/CMakeLists.txt	2022-04-06 12:09:39.243731438 -0400
@@ -24,7 +24,7 @@
         set(EXTERNAL_COMPILE_FLAGS "${EXTERNAL_COMPILE_FLAGS} /WX")
     endif()
 else()
-    set(EXTERNAL_COMPILE_FLAGS "${EXTERNAL_COMPILE_FLAGS} -Werror")
+    #set(EXTERNAL_COMPILE_FLAGS "${EXTERNAL_COMPILE_FLAGS} -Werror")
 endif()
 
 # SHARED
